.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -21,7 +21,8 @@ assignees: ''
 
 ## Don't forget to add labels!
 - component label (see project-specific labels)
-- issue type (`enhancement`, `refactor`, ...)
-- `good first issue` if needed
+- category (`feature` or `enhancement`)
```

Comment:
no `refactor`

![n4go6XXt_400x400](https://github.com/nspcc-dev/.github/assets/24619368/96595479-b514-4142-a877-75685273cff6)

Reply:
No `refactor` -- no problem.

Reply:
i do not agree so i believe i _must_ leave some comment in the PR about it